source/containers.tex

```diff
@@ -3560,6 +3566,51 @@
 Member function \tcode{swap()} shall have a
 non-throwing exception specification.
 
+\rSec3[array.creation]{Array creation functions}
+
+\indexlibrary{\idxcode{array}}%
```

Comment:
Do we want this index entry? This isn't defining `array` here, so the entries for "`array` creation" and `to_array` seem sufficient.